CONTRIBUTING.md

```diff
@@ -215,7 +215,7 @@ Must be one of the following:
 * **docs**: Documentation only changes
 * **style**: Changes that do not affect the meaning of the code (white-space, formatting, missing
   semi-colons, etc)
-* **refactor**: A code change that neither fixes a bug or adds a feature
+* **refactor**: A code change that neither fixes a bug nor adds a feature
```

Comment:
Because https://github.com/angular/angular/blob/master/CONTRIBUTING.md#type
I just copied it.